src/Tribe/Customizer/Global_Elements.php

```diff
@@ -121,59 +172,156 @@ public function get_css_template( $template ) {
+					color: ' . $color_background . ';
+				}
+			';
+
+			$template .= '
+				.tribe-events .tribe-events-calendar-month__mobile-events-icon--event {
+					background-color: <%= global_elements.accent_color %>;
 				}
 			';
 
 			$template .= '
-				.tribe-common .tribe-common-cta--alt:hover,
-				.tribe-common .tribe-common-cta--alt:focus,
-				.tribe-common .tribe-common-cta--thin-alt:hover,
-				.tribe-common .tribe-common-cta--thin-alt:focus,
-				.tribe-theme-twentyseventeen .tribe-common .tribe-common-cta--alt:hover,
-				.tribe-theme-twentyseventeen .tribe-common .tribe-common-cta--alt:focus,
-				.tribe-theme-twentyseventeen .tribe-common .tribe-common-cta--thin-alt:hover,
-				.tribe-theme-twentyseventeen .tribe-common .tribe-common-cta--thin-alt:focus,
-				.tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-cell:not(.tribe-events-calendar-month__day-cell--selected) .tribe-events-calendar-month__day-date,
-				.tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-cell:not(.tribe-events-calendar-month__day-cell--selected) .tribe-events-calendar-month__day-date-link,
 				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-date-link:hover,
 				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-date-link:focus {
-					color: <%= global_elements.accent_color %>;
+					color: ' . $accent_color_hover . ';
 				}
 			';
 
 			$template .= '
-				.tribe-common .tribe-common-cta--thin-alt,
-				.tribe-common .tribe-common-cta--thin-alt:active,
-				.tribe-common .tribe-common-cta--thin-alt:focus,
-				.tribe-common .tribe-common-cta--thin-alt:hover {
-					border-bottom-color: <%= global_elements.accent_color %>;
+				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-date-link:active {
+					color: ' . $accent_color_active . ';
 				}
 			';
 
 			$template .= '
-				.tribe-events .tribe-events-c-ical__link {
-					border-color: <%= global_elements.accent_color %>;
+				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day-cell--selected:hover,
+				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day-cell--selected:focus {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-theme-twentytwenty .tribe-events .tribe-events-calendar-month__day-cell--selected {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec views/full/month/_mobile-events.pcss
+			$template .= '
+				.tribe-events .tribe-events-calendar-month-mobile-events__mobile-event-datetime-featured-text {
 					color: <%= global_elements.accent_color %>;
 				}
 			';
 
+			// overrides for tec views/full/month/_multiday-events.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: d4e05027d570cf63ea51c40c2b492281</sub>